containers/keys/addKey/createKeyHelper.ts

```diff
@@ -0,0 +1,40 @@
+import { OpenPGPKey } from 'pmcrypto';
+import { addKeyAction } from 'proton-shared/lib/keys/keysAction';
+import { getKeyFlagsAddress } from 'proton-shared/lib/keys/keyFlags';
+import getSignedKeyList from 'proton-shared/lib/keys/getSignedKeyList';
+import { createAddressKeyRoute } from 'proton-shared/lib/api/keys';
+import { Address, KeyAction } from 'proton-shared/lib/interfaces';
+
+interface CreateKeyArguments {
+    api: any;
+    fingerprint: string;
+    signingKey: OpenPGPKey;
+    privateKeyArmored: string;
+    keys: KeyAction[];
+    Address: Address;
+}
+export default async ({ api, fingerprint, privateKeyArmored, signingKey, keys, Address }: CreateKeyArguments) => {
+    const updatedKeys = addKeyAction({
+        ID: 'temp',
+        flags: getKeyFlagsAddress(Address, keys),
+        keys,
+        fingerprint
+    });
+
+    const createdKey = updatedKeys.find(({ ID }) => ID === 'temp') as KeyAction;
```

Comment:
not sure if it's a good idea to override undefined checks, maybe just check for undefined before using it?